cvat-cli/src/cvat_cli/_internal/commands_projects.py

```diff
@@ -44,10 +44,42 @@ def configure_parser(self, parser: argparse.ArgumentParser) -> None:
             type=parse_label_arg,
             help="string or file containing JSON labels specification",
         )
+        parser.add_argument(
+            "--dataset_path",
+            default="",
+            type=str,
+            help="path to the dataset file to import",
+        )
+        parser.add_argument(
+            "--dataset_format",
+            default="CVAT 1.1",
+            type=str,
+            help="format of the dataset file being uploaded, e.g. CVAT 1.1",
+        )
+        parser.add_argument(
+            "--completion_verification_period",
+            dest="status_check_period",
+            default=2,
+            type=float,
+            help="period between status checks (only applies when --dataset_path is specified)",
```

Comment:
Could you add `(default: %(default)s)` for all parameters with default values? It will make it more comfortable for users. Some parameters already have it.

Reply:
Okay, updated.